### DISCUSSION TITLE
cbindgen生成的头文件和tquic.h不一样，是不是不能使用cbindgen生成?

### DISCUSSION THREAD

**Discussion by tickmoo** [Tencent/tquic#312] [General]
使用 cbindgen自动生成tquic.h文件：cbindgen --config cbindgen.toml --crate tquic --output include/tquic.h

**Comment by glime-ai** [comment]
要使用 cbindgen 自动生成 `tquic.h` 文件，请执行以下命令：

```sh
cbindgen --config cbindgen.toml --crate tquic --output include/tquic.h
```

这个命令会根据 `cbindgen.toml` 配置文件生成 `tquic.h` 头文件，并将其输出到 `include` 目录下。

This comment was generated by [Glime](https://glimelab.ai/?utm_source=44520fce49ba43798485a628d6928e22&utm_medium=github).

**Comment by iyangsj** [comment]
@tickmoo 目前tquic.h包含少量手工定义代码（例如特殊枚举类型），后续版本这部分会移入到独立文件中

你可以根据diff将新增的差异部分，提交到tquic.h (而不是直接覆盖替换）

**Comment by iyangsj** [comment]
@tickmoo v0.14.0版本已支持
https://github.com/Tencent/tquic/releases
